viewing_party/party.py

```diff
@@ -0,0 +1,199 @@
+def create_movie(title, genre, rating):
+    if title and genre and rating:
+        return {
+                "title": title,
+                "genre": genre,
+                "rating": rating
+        }
+    else:
+        return None
+
+def add_to_watched(user_data , movie):
```

Comment:
👍 

Nice job! Be a little careful of the spacing. We usually don't put a space _before_ a comma
```suggestion
def add_to_watched(user_data, movie):
```